DrDocket/Appointment.h

```diff
@@ -33,8 +36,47 @@ class Appointment {
         string result = buf;
         return result;
     }
-    
- private:
+
+    static void parseDatetime(string datetime, Date& date, Time& time) {
+
+        // Use format: 2016-01-01 10:20
+        replace(datetime.begin(), datetime.end(), '-', ' ');
+        std::replace(datetime.begin(), datetime.end(), ':', ' ');
```

Comment:
Why is this happening twice? Do you still need the std:: ?

Reply:
The first one is replacing dashes, the second is replacing colons.
Maybe there's a way to do them both in one pass (possibly using RegEx) but I'm not too worried about optimizing this part.